test/controllers/term_controller_test.exs

```diff
@@ -117,42 +136,55 @@ defmodule CoursePlanner.TermControllerTest do
     assert html_response(conn, 404)
   end
 
-  test "doesn't show deleted term", %{conn: conn} do
```

Comment:
hmm i mistook assuming we have a seperate test for "doesn't show non-existing term" and I removed this cause it was indirectly related to status. I rename this test and commit it back